packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -227,32 +227,41 @@ class BouncingScrollPhysics extends ScrollPhysics {
   /// The multiple applied to overscroll to make it appear that scrolling past
   /// the edge of the scrollable contents is harder than scrolling the list.
   ///
-  /// By default this is 0.5, meaning that overscroll is twice as hard as normal
-  /// scroll.
-  double get frictionFactor => 0.5;
+  /// This factor starts at 0.54 and progressively becomes harder to overscroll
+  /// as more of the area past the edge is dragged in.
```

Comment:
This seems like the heart of a good change. It might be possible to explain it a little more clearly. Is "inViewFraction" the percentage of scrollable content that's still visible?